magento/pwa-studio
# [bugfix]: Coupon code with leading/trailing whitespace works

## Description

We now trim the whitespace around the entered coupon code.

## Related Issue





Closes [PWA-399](https://jira.corp.magento.com/browse/PWA-399).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add an item to your cart and visit the `/cart` page.
2. Add a valid coupon code but with a leading space. It should work.
3. Remove and add the code but with a trailing space. It should work.
4. Remove and add the code but with a leading _and_ trailing space. It should work.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
